typings/spawk/index.d.ts

```diff
@@ -0,0 +1,130 @@
+declare module "spawk" {
```

Comment:
I made a pull request for this. We can remove it here once that one is merged
https://github.com/wraithgar/spawk/pull/10